stable/fairwinds-insights/Chart.lock

```diff
@@ -1,4 +1,7 @@
 dependencies:
+- name: helm-release-pruner
+  repository: https://charts.fairwinds.com/stable
+  version: 3.2.5
```

Comment:
I think we should keep helm-release-pruner in the insights-release repo if possible. It doesn't really have anything to do with Insights

Reply:
Makes sense